### PR TITLE
PayPal Here credit card reader - basic support

### PR DESCRIPTION
**Pull request by a81j**
Basic support for PayPal Here credit card reader on iOS and Android

### CODE REVIEW

**Comment by retroneo** [comment]
The long-standing major issue still found in v6.24.5  #4686 / #4828 / #4289 is still present.

It would be ideal if payment methods were plugins - more modular.

File: erpnext/public/js/pos/pos.js [hunk 1]
```diff
@@ -505,6 +507,37 @@ erpnext.pos.PointOfSale = Class.extend({
 			me.frm.set_value("write_off_amount", me.frm.doc.grand_total - paid_amount);
 			me.frm.set_value("outstanding_amount", 0);
 
+            if (is_paypal_here) {
+                var invoice = {
+                    "merchantEmail": "test@test.com",
```
Comment (anandpdoshi) [inline erpnext/public/js/pos/pos.js]: Should not be hardcoded

**Comment by anandpdoshi** [comment]
> The long-standing major issue still found in v6.24.5 #4686 / #4828 / #4289 is still present.
Thanks for pointing out these issues. I've fixed this issue.

**Comment by rmehta** [comment]
Again POS payment extensions can be moved to a separate app. @saurabh6790

**Pull request closed without merging**